frontend/app/component/election/ElectionStatusWithIcon.tsx

```diff
@@ -0,0 +1,47 @@
+import { ElectionStatus } from "@kiesraad/api";
+import { t } from "@kiesraad/i18n";
+import { IconCheckHeart, IconCheckVerified, IconLock } from "@kiesraad/icon";
+import { Icon } from "@kiesraad/ui";
+
+function getIconForElectionStatus(status: ElectionStatus) {
+  switch (status) {
+    case "DataEntryInProgress":
+      return <Icon size="md" color="accept" icon={<IconCheckHeart />} />;
+    case "FinishDataEntry":
+      return <Icon size="md" color="error" icon={<IconLock />} />;
+    case "DataEntryFinished":
+      return <Icon size="md" color="default" icon={<IconCheckVerified />} />;
```

Comment:
For this `case` I also have the button "Invoerfase afronden" on the screen. That seems inconsistent.

![image](https://github.com/user-attachments/assets/29bda925-3c38-4ba4-a951-cab80fc0c502)

Reply:
Fixed in b514939d3a57324737f2b1e833db436db664ad6d